新增项目导入和清空

bshy522 opened the pull request:
<img width="1562" height="546" alt="CleanShot 2025-12-24 at 17 50 13@2x" src="https://github.com/user-attachments/assets/937721b2-3384-4f58-86cc-3fee0b5a9740" />

Commit:
feat: add support for importing local directories to OPFS

Commit:
feat: add clear all files functionality

Commit:
feat: add TypeScript types for wicg-file-system-access and improve import directory functionality

Commit:
Merge remote-tracking branch 'origin/main' into feat/project-importer

* origin/main:
  feat: enable logFilter
  Fix build error: remove invalid validate property from html options
  Fix build error: remove unknown property eol
  Fix build error: remove unknown property wrapAttributesIndentSize
  Disable syntax validation in Monaco editor
  refactor: remove unused webpack copy plugin and fallback configs (#9)
  feat: add tailwind css support and update demo files (#7)

elrrrrrrr commented on app/hooks/useImportDirectory.ts:
```diff
@@ -0,0 +1,108 @@
+
+			toast.success(`Imported ${selectedDirHandle.name} successfully`);
+			refresh();
+			await installDependencies(project);
```
😄 参照下 utooland/utoo 里的 deps 流程？ 可以不用再手动传 package-lock.json 了

bshy522 replied:
如果有的话 还是用已有的吧，感觉更能还原问题

Commit:
chore: update dependencies and improve error handling in useFileContent hook

- Bump versions of several dependencies in package.json and pnpm-lock.yaml, including @monaco-editor/react, @radix-ui/react-progress, and tailwind-merge.
- Enhance error handling in the useFileContent hook by replacing setError with toast notifications for better user feedback.
- Add "src/app.tsx" to the initial state of openFiles in the useFileContent hook.

Commit:
chore: add .npmrc file and remove pnpm-lock.yaml

- Introduced a new .npmrc file with the setting `lockfile=false` to disable lockfile generation.
- Removed the pnpm-lock.yaml file to streamline dependency management.
